graspologic/match/wrappers.py

```diff
@@ -0,0 +1,274 @@
+        connectomes," bioRxiv 2022.05.19.492713 (2022)
+
+    .. [3] D. Fishkind, S. Adali, H. Patsolic, L. Meng, D. Singh, V. Lyzinski, C. Priebe,
+        Seeded graph matching, Pattern Recognit. 87 (2019) 203–215
+
+    .. [4] A. Saad-Eldin, B.D. Pedigo, C.E. Priebe, J.T. Vogelstein "Graph Matching via
+       Optimal Transport," arXiv 2111.05366 (2021)
+
+    Returns
+    -------
+    res: MatchResult
+        ``MatchResult`` containing the following fields.
+
+        indices_A : ndarray
+            Indices in ``A`` which were matched.
+
+        indices_B : ndarray
+            Indices in ``B`` which were matched.
+
+        score : float
+            Objective function value at the end of optimization
+
+        misc : list of dict
+            List of length ``n_init`` containing information about each run. Fields for
+            each run are ``score``, ``n_iter``, ``convex_solution``, and ``converged``.
+    """
+
+    if use_numba:
+        raise NotImplementedError("Still working on numba implementation")
+
+    max_seed = np.iinfo(np.uint32).max
+
+    if (rng is not None) and (not isinstance(rng, np.random.Generator)):
+        rng = check_scalar(rng, "rng", (int, np.integer), min_val=0, max_val=max_seed)
+    # otherwise the input is None or a random Generator - these can be passed in to
+    # default_rng safely
+
+    rng = np.random.default_rng(rng)
+
+    seeds = rng.integers(max_seed, size=n_init)
+
+    if n_init > 1:
+        parallel_verbose = verbose
+        solver_verbose: Int = 0
+    else:
+        parallel_verbose = 0
+        solver_verbose = verbose
+
+    solver = GraphMatchSolver(
+        A=A,
+        B=B,
+        AB=AB,
+        BA=BA,
+        S=S,
+        partial_match=partial_match,
+        init=init,
+        init_perturbation=init_perturbation,
+        verbose=solver_verbose,
+        shuffle_input=shuffle_input,
+        padding=padding,
+        maximize=maximize,
+        max_iter=max_iter,
+        tol=tol,
+        use_numba=use_numba,
+        transport=transport,
+        transport_regularizer=transport_regularizer,
+        transport_tol=transport_tol,
+        transport_max_iter=transport_max_iter,
+    )
+
+    def run_single_graph_matching(seed: RngType) -> MatchResult:
+        solver.solve(seed)
+        matching = solver.matching_
+        indices_A = matching[:, 0]
+        indices_B = matching[:, 1]
+        score = solver.score_
+        misc: Dict[str, Any] = {}
+        misc["score"] = score
+        misc["n_iter"] = solver.n_iter_
+        misc["convex_solution"] = solver.convex_solution_
+        misc["converged"] = solver.converged_
+        return MatchResult(indices_A, indices_B, score, misc)
+
+    seeds = rng.integers(max_seed, size=n_init)
+    parallel = Parallel(n_jobs=n_jobs, verbose=parallel_verbose)
+    results = parallel(delayed(run_single_graph_matching)(seed) for seed in seeds)
+    # results = [run_single_graph_matching(seeds)]
```

Comment:
if this isn't used anymore, let's remove it?

Reply:
thanks, i'll be sure to clean up all hanging comments like that

Reply:
✅ https://github.com/microsoft/graspologic/pull/960/commits/eac08772d81b0afbf8f84f95e3e32c3cc24ae091